lib/ansible/modules/cloud/vmware/vmware_vswitch.py

```diff
@@ -138,7 +150,11 @@ def state_create_vswitch(self):
         vss_spec.numPorts = self.number_of_ports
         vss_spec.mtu = self.mtu
         if self.nic_name:
-            vss_spec.bridge = vim.host.VirtualSwitch.BondBridge(nicDevice=[self.nic_name])
+            if isinstance(self.nic_name, list):
+                # Multiple Physical NICs
+                vss_spec.bridge = vim.host.VirtualSwitch.BondBridge(nicDevice=self.nic_name)
+            else:
+                vss_spec.bridge = vim.host.VirtualSwitch.BondBridge(nicDevice=[self.nic_name])
```

Comment:
These 5 lines could be reduced to just this:
```python
vss_spec.bridge = vim.host.VirtualSwitch.BondBridge(nicDevice=self.nic_name)
```
Because it should already be a list.